mne_bids/write.py

```diff
@@ -187,14 +189,17 @@ def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
                             ('z', z),
                             ('size', sizes),
                             ])
-    else:
+    elif kind == 'eeg':
         data = OrderedDict([('name', names),
                             ('x', x),
                             ('y', y),
                             ('z', z),
                             ])
+    else:  # pragma: no cover
+        raise RuntimeError("kind {} not supported.".format(kind))
+
     _write_tsv(fname, data, overwrite=overwrite, verbose=verbose)
-    return fname
+    return fname  # XXX why return parameter???
```

Comment:
I think this should be deleted @agramfort

Reply:
you agree that there is no point in returning fname?
_electrodes_tsv does the same and I have no clue why

Reply:
yes agreed

Reply:
I think that's the same for all the _sidecar writing functions then?

Reply:
@sappelhoff any idea why this is like that?

Reply:
Deleted in _electrodes and _coordsystem

Reply:
I think it was convenient at some point? But I also don't remember :shrug: